src/MessageManager.cpp

```diff
@@ -207,7 +215,7 @@ void MessageManager::Unsubscribe( IMessageSubscriber* pSubscriber, MessageID m )
 void MessageManager::Broadcast( Message &msg ) const
 {
 	// GAMESTATE is created before MESSAGEMAN, and has several BroadcastOnChangePtr members, so they all broadcast when they're initialized.
-	if(m_Logging)
+	if(this != nullptr && m_Logging)
```

Comment:
...wow. Whoops.

Yeah, we'll have to make our code more robust to avoid needing the check.